src/main/guidance/chatWidget/useChatGuidanceCommunication.ts

```
@@ -0,0 +1,102 @@
+    const introMessage = {
+      id: '__intro',
+      createdAt: new Date(),
+      reactions: [],
+      message: t('chatbot.intro'),
+      author: undefined,
+    };
+
+    if (messagesData?.lookup.room?.messages.length) {
+      return [
+        introMessage,
+        ...messagesData?.lookup.room?.messages?.map(message => ({
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)